crates/languages/src/python.rs

```diff
@@ -649,6 +652,29 @@ fn env_priority(kind: Option<PythonEnvironmentKind>) -> usize {
     }
 }
 
+/// Return the name of environment declared in <worktree-root/.venv.
+///
+/// https://virtualfish.readthedocs.io/en/latest/plugins.html#auto-activation-auto-activation
+fn get_worktree_venv_declaration(worktree_root: &Path) -> Option<String> {
+    let dot_venv = worktree_root.join(".venv");
+
+    if fs::metadata(&dot_venv)
```

Comment:
Can we forego call to fs::metadata? Subsequent file open will fail anyways if dot_env is not a file/does not exist.
This way we can slash # of syscalls per worktree_root.
The only caveat is that we'll probably want to strip that log::info call.

Reply:
Done